albumentations/core/transforms_interface.py

```diff
@@ -95,7 +95,11 @@ def targets(self):
     def update_params(self, params, **kwargs):
         if hasattr(self, 'interpolation'):
             params['interpolation'] = self.interpolation
-        params.update({'cols': kwargs['image'].shape[1], 'rows': kwargs['image'].shape[0]})
+        if hasattr(self, 'fill_value'):
+            params['interpolation'] = self.fill_value
```

Comment:
it's strange to set interpolation as fill value

Reply:
Not strange, just plain incorrect! That's what I get for copy-pasting. I
fixed it and updated the PR.